bonobo/ext/console.py

```diff
@@ -79,7 +79,7 @@ def write(self, context, prefix='', rewind=True, append=None):
             else:
                 _line = ''.join(
                     (
-                        ' ', Fore.BLACK, '-', ' ', node.name, name_suffix, ' ', node.get_statistics_as_string(),
+                        ' ', Fore.GREEN, '-', ' ', node.name, name_suffix, ' ', node.get_statistics_as_string(),
```

Comment:
I see this is outdated compared to develop branch which is using `dead_color = (Style.BRIGHT + Fore.BLACK) if self.iswindows else Fore.BLACK`. Can you test if `Style.BRIGHT + Fore.BLACK` is an okay-ish color for you on ubuntu default terminal ? That would allow to simplify and have the same color everywhere. Thanks.